README.md

```diff
@@ -44,7 +46,7 @@ To run an agent with a custom prompt, simply do this `python beeai_framework_sta
 
 ## 🔒Code interpreter
 
-The [Bee Code Interpreter](https://github.com/i-am-bee/bee-code-interpreter) is a gRPC service that an agent uses to execute an arbitrary Python code safely.
+The [Bee Code Interpreter](https://github.com/i-am-bee/bee-code-interpreter) is an HTTP service that an agent uses to execute an arbitrary Python code safely.
```

Comment:
the **About** section of [the repo](https://github.com/i-am-bee/bee-code-interpreter) mentions it is an HTTP service